src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -172,6 +172,23 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
         t.transactionResponse.map(Extraction.decompose).getOrElse(JNull)
     }
 
+  implicit val eth_getRawTransactionByBlockHashAndIndex =
```

Comment:
It might make sense to reduce responses to single type and create single response encoder. WDYT?

Reply:
responses are unified: RawTransactionResponse: https://github.com/input-output-hk/mantis/pull/720/commits/973fbed4615a5192ce6abac3d88e66537584f3b2
BTW that revealed bugs in handling responses!

There are pairs of requests that have the same parameters (and the same meaning):
eth_getRawTransactionByBlockHashAndIndex
eth_getTransactionByBlockHashAndIndex
etc so I unify them as well: https://github.com/input-output-hk/mantis/pull/720/commits/586f9a591d40ed5f8857227f9a55eaee489d8b9c